jdi-light-mobile-tests/src/test/java/io/github/epam/html/tests/elements/composite/page/init/PageInitTests.java

```diff
@@ -0,0 +1,83 @@
+package io.github.epam.html.tests.elements.composite.page.init;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import page.objects.jdi.HomePageWeb;
+import page.objects.jdi.HomePageWebEx;
+import page.objects.selenium.HomePage;
+import page.objects.selenium.HomePageEx;
+
+import static com.epam.jdi.light.elements.composite.WebPage.*;
+import static com.epam.jdi.light.elements.init.UIFactory.*;
+import static page.objects.selenium.PageBase.*;
+
+public class PageInitTests {
+    static String expecterdUrl(String url) {
+        return JDI_DOMAIN + url;
+    }
+    boolean first = true;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5217753)